core/templates/filters/convert-html-to-unicode.filter.ts

```diff
@@ -16,8 +16,12 @@
  * @fileoverview Converts HTML to unicode.
  */
 
+export const convertHtmlToUnicode = (html: string): string => {
+  return angular.element('<div>' + html + '</div>').text();
+};
```

Comment:
Can this be made into a regular function definition?

```suggestion
export convertHtmlToUnicode(html: string): string {
  return angular.element('<div>' + html + '</div>').text();
}
```

Reply:
@brianrodri the linter doesn't allow me to create a regular function as you suggested here.
![functionError](https://user-images.githubusercontent.com/31998550/96878269-429b7180-1498-11eb-8d00-6b775ccbdada.png)

Reply:
Hmm that expression looks wrong, have you tried using the exact suggesstion I provided?

Reply:
Yup, still the same issue.
![functionErrorBrian](https://user-images.githubusercontent.com/31998550/96903095-045f7b80-14b3-11eb-93e1-6dd4839da5c3.png)

Reply:
Yup still the same issue. The linter wants to export a declaration.
![2020-10-22-220851_1920x1080_scrot](https://user-images.githubusercontent.com/31998550/96903341-5902f680-14b3-11eb-9959-2dfac456f74f.png)

Reply:
OK, fine with me then.